oap-server/server-storage-plugin/storage-jdbc-hikaricp-plugin/src/main/java/org/apache/skywalking/oap/server/storage/plugin/jdbc/mysql/MySQLStorageConfig.java

```diff
@@ -39,5 +39,18 @@
      * @since 8.2.0
      */
     private int numOfSearchableValuesPerTag = 2;
+    /**
+     * max size per batch execute sql
+     */
+    private int maxSizeOfBatchSql = 2000;
+    /**
+     * async batch execute pool size
+     */
+    private int h2AsyncBatchPersistentPoolSize = 4;
+    /**
+     * async batch execute channel size
+     */
+    private int h2AsyncBatchPersistentChannelSize = 4;
+
```

Comment:
Same as above, you should consider merging. And `h2` prefix is incorrect.

Reply:
ok. i use asyncBatchPersistentPoolSize  and remove h2AsyncBatchPersistentChannelSize